internal/pgmonitor/exporter.go

```diff
@@ -152,12 +152,9 @@ func ExporterStartCommand(builtinCollectors bool, commandFlags ...string) []stri
 		`  echo "Starting postgres_exporter with the following flags..."`,
 		`  echo "${postgres_exporter_flags[@]}"`,
 		`  postgres_exporter "${postgres_exporter_flags[@]}" &`,
-		`  echo $! > $POSTGRES_EXPORTER_PIDFILE`,
+		`  [ -e /proc/$! ] && echo $! > $POSTGRES_EXPORTER_PIDFILE`,
 		`}`,
 
-		// run function to combine queries files and start postgres_exporter
-		`start_postgres_exporter`,
```

Comment:
🤔 It seems reasonable to me to start it up when the entrypoint first runs. Why remove it?